packages/locale/lang/ar-eg.ts

```diff
@@ -0,0 +1,147 @@
+    },
+    datepicker: {
+      now: 'الآن',
+      today: 'اليوم',
+      cancel: 'إلغاء',
+      clear: 'إزالة',
+      confirm: 'موافق',
+      dateTablePrompt:
+        'استخدم مفاتيح الاسهم و اضغط انتر لاختيار اليوم المراد من الشهر',
+      monthTablePrompt: 'استخدم مفاتيح الاسهم واضغط انتر لاختيار الشهر',
+      yearTablePrompt: 'استخدم مفاتيح الاسهم واضغط انتر لاختيار السنة',
+      selectDate: 'إختر التاريخ',
+      selectTime: 'إختر الوقت',
+      startDate: 'تاريخ البدء',
+      startTime: 'وقت البدء',
+      endDate: 'تاريخ الإنتهاء',
+      endTime: 'وقت الإنتهاء',
+      prevYear: 'السنة السابقة',
+      nextYear: 'السنة التالية',
+      prevMonth: 'الشهر السابق',
+      nextMonth: 'الشهر التالي',
+      year: 'سنة',
+      month1: 'يناير',
+      month2: 'فبراير',
+      month3: 'مارس',
+      month4: 'أبريل',
+      month5: 'مايو',
+      month6: 'يونيو',
+      month7: 'يوليو',
+      month8: 'أغسطس',
+      month9: 'سبتمبر',
+      month10: 'أكتوبر',
+      month11: 'نوفمبر',
+      month12: 'ديسمبر',
+      week: 'أسبوع',
+      weeks: {
+        sun: 'الأحد',
+        mon: 'الأثنين',
+        tue: 'الثلاثاء',
+        wed: 'الأربعاء',
+        thu: 'الخميس',
+        fri: 'الجمعة',
+        sat: 'السبت',
+      },
+      months: {
+        jan: 'يناير',
+        feb: 'فبراير',
+        mar: 'مارس',
+        apr: 'أبريل',
+        may: 'مايو',
+        jun: 'يونيو',
+        jul: 'يوليو',
+        aug: 'أغسطس',
+        sep: 'سبتمبر',
+        oct: 'أكتوبر',
+        nov: 'نوفمبر',
+        dec: 'ديسمبر',
+      },
+    },
+    inputNumber: {
+      decrease: 'طرح رقم',
+      increase: 'زيادة رقم',
+    },
+    select: {
+      loading: 'جار التحميل',
+      noMatch: 'لايوجد بيانات مطابقة',
+      noData: 'لايوجد بيانات',
+      placeholder: 'إختر',
+    },
+    dropdown: {
+      toggleDropdown: 'تبديل القائمة',
+    },
+    cascader: {
+      noMatch: 'لايوجد بيانات مطابقة',
+      loading: 'جار التحميل',
+      placeholder: 'إختر',
+      noData: 'لايوجد بيانات',
+    },
+    pagination: {
+      goto: 'أذهب إلى',
+      pagesize: '/صفحة',
+      total: 'الكل {total}',
+      pageClassifier: '',
+      page: 'صفحة', // to be translated
+      prev: 'انتقل إلى الصفحة السابقة', // to be translated
+      next: 'انتقل إلى الصفحة التالية', // to be translated
+      currentPage: 'صفحة رقم {pager}', // to be translated
+      prevPages: 'صفحات {pager} السابقة', // to be translated
+      nextPages: 'صفحات {pager} التالية', // to be translated
+    },
+    dialog: {
+      close: 'أغلق هذا التبويب',
+    },
+    drawer: {
+      close: 'أغلق هذا التبويب',
+    },
+    messagebox: {
+      title: 'العنوان',
+      confirm: 'موافق',
+      cancel: 'إلغاء',
+      error: 'مدخل غير صحيح',
+      close: 'أغلق هذا التبويب',
+    },
+    upload: {
+      deleteTip: 'اضغط ازالة لحذف المحتوى',
+      delete: 'حذف',
+      preview: 'عرض',
+      continue: 'إستمرار',
+    },
+    table: {
```

Comment:
Missing translations for the _**slider**_ component.
https://github.com/element-plus/element-plus/blob/main/packages/locale/lang/en.ts#L131-L135

Reply:
Done

Reply:
Do you forget to commit something? The slider component doesn't still get translated.

Reply:
Kindly check again